clients/wallet_daemon_client/src/types.rs

```diff
@@ -108,6 +108,11 @@ pub struct TransactionGetRequest {
     pub hash: FixedHash,
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct TransactionGetAllRequest {
```

Comment:
Can we rename the struct to `GetAllTransactionsRequest` or `GetAllTransactionsByStatusRequest` ?  I think we use `Get` at the beginning in the previous examples.

Reply:
I want to keep it consistent with `TransactionGetRequest` `TransactionSubmitRequest` etc.